utils/python/tests/test_config_build.py

```diff
@@ -0,0 +1,581 @@
+#!/usr/bin/env python
+"""Tests for the generation of Macros files from config_build.xml"""
+
+# The invalid-name check is not very useful, while line-too-long and
+# bad-continuation pick up on a lot of lines that are hard to split up
+# legibly because they contain long strings.
+# pylint: disable=invalid-name,line-too-long,bad-continuation
```

Comment:
OK. What's the preferred way to do this? Do you want me to copy it into the same file, or turn this into a module loaded by `scripts_regression_tests`?